libs/ui-lib/lib/cim/components/ClusterDeployment/ClusterDeploymentReviewStep.tsx

```diff
@@ -143,6 +146,26 @@ const ClusterDeploymentReviewStep = ({
             value={<ReviewHostsInventory hosts={cluster.hosts} />}
           />
 
+          {!!customManifestNames.length ? (
+            <DetailItem
+              title={t('ai:Custom manifests')}
+              testId="custom-manifests"
+              value={
+                <Table variant={TableVariant.compact} borders={false}>
+                  <Tbody>
+                    {customManifestNames.map((name, id) => (
```

Comment:
the `name` prop as key should be enought. AFAIK it needs to be unique (we should also add that validation to our form)